index.html

```diff
@@ -8,5 +8,6 @@
 </head>
 <body>
     <h1>Hello</h1>
+    <h2>THis is line added newly</h2>
 </body>
```

Comment:
This could be written in a much better way

Reply:
ok i got it